tests/index.js

```diff
@@ -119,6 +119,66 @@ module.exports = function (test, Store) {
     })
   })
 
+  test('get with null option', function (t) {
+    var store = new Store(10)
+    store.put(0, new Buffer('0123456789'), function (err) {
+      t.error(err)
+      store.get(0, null, function (err, chunk) {
+        t.error()
```

Comment:
`t.error(err)` (also in the existing code and the additional cases below)

Reply:
Good catch!